python/scenario/_events/event_reporter.py

```diff
@@ -64,7 +64,7 @@ async def post_event(self, event: ScenarioEvent) -> Dict[str, Any]:
             return result
 
         try:
-            async with httpx.AsyncClient() as client:
+            async with httpx.AsyncClient(follow_redirects=True) as client:
```

Comment:
🤔 why do we have redirects?

Reply:
@rogeriochaves No idea. Seems new. Cloudflare stuff?